.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,70 @@
+name: Bug Report
+description: Use this to report any bugs you have encountered in game.
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Confirmations
+      description: Please confirm the below before continuing
+      options:
+        - label: I have searched open and closed issues for any existing report of my issue
+          required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: What happened?
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: What did you expect to happen?
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to Reproduce
+      description: Describe how we can make this bug happen ourselves.
+      placeholder: |
+        1. Go to '...'
+        2. Click on '...'
+        3. Scroll down to '...'
+        4. See error
+    validations:
+      required: true
+  - type: input
+    id: os
+    attributes:
+      label: What operating system are you using?
+      placeholder: e.g. Windows 10, iOS, Android
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: What browsers are you seeing the problem on?
+      multiple: true
+      options:
+        - Firefox
+        - Chrome
```

Comment:
Missing "Opera" (have seen a lot of bugreports with this) and one saying just "other"